src/hotspot/share/utilities/preserveException.cpp

```diff
@@ -23,59 +23,34 @@
  */
 
 #include "precompiled.hpp"
+#include "memory/resourceArea.hpp"
+#include "oops/oop.hpp"
 #include "runtime/handles.inline.hpp"
 #include "utilities/preserveException.hpp"
 
-// TODO: These three classes should be refactored
-
-PreserveExceptionMark::PreserveExceptionMark(Thread*& thread) {
-  thread     = Thread::current();
+PreserveExceptionMark::PreserveExceptionMark(Thread* thread) {
   _thread    = thread;
```

Comment:
Nit: extra spaces before = now that lines are not being aligned.

Reply:
fixed.